test/uxCapture.test.js

```diff
@@ -153,15 +157,12 @@ describe('UXCapture', () => {
 	});
 
 	describe('mark', () => {
-		beforeAll(() => {
-			UXCapture.create({ onMark, onMeasure });
-		});
-
 		beforeEach(() => {
 			ExpectedMark.clearExpectedMarks();
 			onMark.mockClear();
 			onMeasure.mockClear();
 
+			UXCapture.create({ onMark, onMeasure });
```

Comment:
Same thing here, the `UXCapture.create()` interface is meant to be called once. I'd leave it in `beforeAll()` where it will get executed only a single time before all the individual tests are run.

Reply:
Yeah, about that - it's supposed to be called once per page view.

For tests that test page views we might want to call it before each test.

Unless we relax the requirement that startView can be called without calling startTransition().

Reply:
I'm not sure I understand - I was expecting that `UXCapture.create` should be called once when the application initializes, before anything view-related happens, but then it's never called again.

`startView` is called on initial page render and then subsequently only after `startTransition` - I think that's a hard requirement for startView

Reply:
Yeah, I'm not sure I follow either. If you need help with this let me know! I've spent a lot of time in unit tests. :)

Reply:
You know what, maybe I should've just done startTransition before each test rather than create which should be happening once. Let me try that and see how that goes.

Otherwise I might need to pair with @sadafie to wrap my brain around that.

And I agree, @mmcgahan, startTransition is a hard requirement which I'd rather not relax.

Reply:
Adding startTransition() calls in beforeEach() worked so UXCapture.create() is back in beforeAll()